Fun.Build/StageContextExtensions.fs

```diff
@@ -205,6 +205,8 @@ module StageContextExtensionsInternal =
 
                     let extraInfo = $"timeout: {timeoutForStage}ms. step timeout: {timeoutForStep}ms."
                     match index with
+                    | StageIndex.WhenStage ->
+                        AnsiConsole.Write(Rule($"[grey50]WHEN STAGE [bold turquoise4]{namePath}[/] started. {extraInfo}[/]").LeftJustified())
```

Comment:
Also change **WHEN STAGE** to "CONDITIONAL STAGE"

Reply:
Would you be ok with "CONDITION STAGE"?

"CONDITIONAL STAGE" means that this stage may or may not run. But actually this stage will always run as a condition for other stages.